MoltenVKShaderConverter/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+##
+## MoltenVKShaderConverter Files
+##
+
+add_library(moltenvk_shader_converter STATIC
+        ../Common/MVKOSExtensions.mm
+        MoltenVKShaderConverter/SPIRVSupport.cpp
+        MoltenVKShaderConverterTool/MoltenVKShaderConverterTool.cpp
+        MoltenVKShaderConverterTool/OSSupport.mm
+        MoltenVKShaderConverterTool/main.cpp
+        MoltenVKShaderConverter/FileSupport.mm
+        MoltenVKShaderConverter/GLSLConversion.mm
+        MoltenVKShaderConverter/GLSLToSPIRVConverter.cpp
+        MoltenVKShaderConverter/SPIRVConversion.mm
+        MoltenVKShaderConverter/SPIRVToMSLConverter.cpp
+        )
+
+find_library(FOUNDATION_LIBRARY Foundation)
+message(STATUS "FOUNDATION_LIBRARY is ${FOUNDATION_LIBRARY}")
```

Comment:
Why are you printing the framework paths? Also why not use REQUIRED on the find_package calls, so that CMake errors when it can't find them?